packages/data-context/src/actions/WizardActions.ts

```diff
@@ -63,6 +79,105 @@ export class WizardActions {
     return this.data
   }
 
+  async initialize () {
+    if (this.ctx.currentProject) {
+      this.data.detectedFramework = null
+      this.data.detectedBundler = null
+      this.data.detectedLanguage = null
+
+      await this.detectLanguage()
+      debug('detectedLanguage %s', this.data.detectedLanguage)
+      this.data.chosenLanguage = this.data.detectedLanguage || 'js'
+
+      let hasPackageJson = true
+
+      try {
+        await this.ctx.fs.access(path.join(this.ctx.currentProject, 'package.json'), this.ctx.fs.constants.R_OK)
+      } catch (e) {
+        debug('Could not read or find package.json: %O', e)
+        hasPackageJson = false
+      }
+      const packageJson: {
+        dependencies?: { [key: string]: string }
+        devDependencies?: { [key: string]: string }
+      } = hasPackageJson ? await this.ctx.fs.readJson(path.join(this.ctx.currentProject, 'package.json')) : {}
+
+      debug('packageJson %O', packageJson)
+      const dependencies = [
+        ...Object.keys(packageJson.dependencies || {}),
+        ...Object.keys(packageJson.devDependencies || {}),
+      ]
+
+      this.detectFramework(dependencies)
+      debug('detectedFramework %s', this.data.detectedFramework)
+      this.detectBundler(dependencies)
+      debug('detectedBundler %s', this.data.detectedBundler)
+
+      this.data.chosenFramework = this.data.detectedFramework || null
+      this.data.chosenBundler = this.data.detectedBundler || null
+    }
+  }
+
+  private detectFramework (dependencies: string[]) {
+    // Detect full featured frameworks
+    if (dependencies.includes('next')) {
+      this.ctx.wizardData.detectedFramework = 'nextjs'
+    } else if (dependencies.includes('react-scripts')) {
+      this.ctx.wizardData.detectedFramework = 'cra'
+    } else if (dependencies.includes('nuxt')) {
+      this.ctx.wizardData.detectedFramework = 'nuxtjs'
+    } else if (dependencies.includes('@vue/cli-service')) {
+      this.ctx.wizardData.detectedFramework = 'vuecli'
+    }
+
+    if (this.ctx.wizardData.detectedFramework) {
```

Comment:
Any reason why this is between the first and second lot of dependency check conditionals?

Reply:
none anymore (it used to be where I set up the bundler)